config.json

```diff
@@ -922,7 +940,7 @@
         "name": "Spiral Matrix",
         "uuid": "c1abafcc-0d44-4fb5-afae-bff3ce2e1b39",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays", "for-loops", "conditionals"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/spiral-matrix/solutions/TomPradat

Most solutions don't use switch conditionals so I didn't add it